src/app/features/test-records/test-records-routing.module.ts

```diff
@@ -4,13 +4,32 @@ import { NoEditGuard } from '@guards/no-edit.guard';
 import { TestResultResolver } from 'src/app/resolvers/test-result/test-result.resolver';
 import { AmendedTestRecordComponent } from './views/amended-test-record/amended-test-record.component';
 import { TestRecordComponent } from './views/test-record/test-record.component';
+import { TestSectionsComponent } from './views/test-sections/test-sections.component';
+import { TestComponent } from './views/test/test.component';
 
 const routes: Routes = [
   {
-    path: ':systemId/test-result/:testResultId',
+    path: ':systemId/test-result/:testResultId/:testTypeId',
     component: TestRecordComponent,
     data: { title: 'Test Result' },
     resolve: { load: TestResultResolver }
+    // children: [
```

Comment:
Is there a reason this left in commented out?

Reply:
This is in case we want to go back to using tabbed view for each section/seeing only one section at a time. Would save work if/when we want to make prototypes for service design team to demo to users. Would clean it up when it comes to merging into `develop`